flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/EICAS_Common.js

```diff
@@ -186,5 +189,9 @@ class EICASCommonDisplay extends Airliners.EICASTemplateElement {
         }
         return payloadWeight;
     }
+    refreshHomeCockpitMode() {
+        const isHomeCockpit = SimVar.GetSimVarValue("L:A32NX_HOME_COCKPIT_ENABLED", "bool");
+        this.backlightBleed.style.visibility = (isHomeCockpit === 1) ? "hidden" : "visible";
```

Comment:
**issue:** style property is not diffed

Reply:
Could you elaborate what you mean by diffed? You mean capture the initial visibility value prior to performing any change?